Az.DevTestLabs2.psm1

```diff
@@ -243,8 +238,11 @@ function DeployLab {
       Enable-AzureRmAlias -Scope Local -Verbose:$false
 
       # WORKAROUND
-      Disable-AzContextAutosave -Scope Process | Out-Null
-      Import-AzContext -Path (Join-Path $env:TEMP "AzContext.json") | Out-Null
+      $Mutex = New-Object -TypeName System.Threading.Mutex -ArgumentList $false, "Global\AzDtlLibrary"
```

Comment:
This is a workaround to avoid checking the saved azure context with more than 1 thread at a time..  Details are in the bug listed here:  https://github.com/Azure/azure-powershell/issues/9448  . Once this is fixed we can remove the workarounds.